scripts/build.py

```diff
@@ -1926,7 +1926,11 @@ def download(obj, path):
         # * Editor files
         # * Defold SDK files
         # * launcher files, used to launch editor2
-        pattern = re.compile(r'(^|/)editor(2)*/|/defoldsdk\.zip$|/launcher(\.exe)*$')
+        # * rarely used platforms: armv7-android , js-web  and x86-win32
+        # * headless builds
```

Comment:
We don't include headless for a long time, so we don't need to download them when bundling bob as well as excluded platforms